doc/configuration.md

```diff
@@ -105,5 +76,35 @@ following section.
     - **name** (string): the name of the unit. This must be suffixed with a
                          valid unit type (e.g. "00-eth0.network").
     - **contents** (string): the contents of the unit.
+- **passwd** (object): describes the desired additions to the passwd database.
+  - **users** (list of objects): the list of accounts to be added.
+    - **name** (string): the username for the account.
+    - **passwordHash** (string): the encrypted password of the new account.
+    - **sshAuthorizedKeys** (list of strings): a list of SSH keys to be added
+                                               to the user's authorized_keys.
+    - **create** (object): contains the set of options to be used when creating
+                           the user. A non-null entry indicates that the user
+                           account shall be created.
+      - **uid** (integer): the user ID of the new account.
+      - **gecos** (string): the GECOS field of the new account.
+      - **homeDir** (string): the home directory of the new account.
+      - **noCreateHome** (boolean): whether or not to create the user's home
+                                    directory.
+      - **primaryGroup** (string): the name or ID of the primary group of the
+                                   new account.
+      - **groups** (list of strings): the list of supplementary groups of the
+                                      new account.
+      - **noUserGroup** (boolean): whether or not to create a group with the
+                                   same name as the user.
+      - **system** (boolean): whether or not to make this account a system
+                              account.
+      - **noLogInit** (boolean): whether or not to add the user to the lastlog
+                                 and faillog databases.
+      - **shell** (string): the login shell of the new account.
+  - **groups** (list of objects): the list of groups to be added.
+    - **name** (string): the name of the group.
+    - **gid** (integer): the group ID of the new group.
+    - **passwordHash** (string): the encrypted password of the new group.
+    - **system** (boolean): whether or not to make this group a system group.
```

Comment:
pretty sure we explicitly removed this option from cloudinit, why is it in ignition?